courses/programming-in-haskell/pih-exercises/src/Chapter07.hs

```diff
@@ -0,0 +1,17 @@
+-- | Exercises for chapter 7.
+
+module Chapter07 (comprehension, map') where
```

Comment:
the Prelude.map is needed for Exercise 2
I could not get that to work when hiding